.github/workflows/publishVSCode.yml

```diff
@@ -0,0 +1,107 @@
+name: VSCode Publish
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        required: true
+    outputs:
+      result:
+        description: Set to published if the push to the VSCode marketplace was successful
+        value: ${{ jobs.publish.outputs.result }}
+      status: 
+        description: Set to complete if all tasks assoicated with the workflow were successful.
```

Comment:
note we have two outputs here...one to specify if we made it to the marketplace..the other to indicate a fully successful run.

Reply:
unset I believe.  This is all new territory so I'm not 100% clear on how it behaves.